newsfragments/2779.misc.rst

```diff
@@ -0,0 +1 @@
+Extends policy probationary period until October 30th, 2021. No policies may be created on the network beyond this date. 
```

Comment:
Since October has 31 days.

```suggestion
Extends policy probationary period until October 31st, 2021. No policies may be created on the network beyond this date. 
```

Reply:
✔️